content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -151,85 +168,149 @@ The next step is to sign up for Prisma's [Data Proxy](/data-platform/data-proxy)
 
-The Data Proxy Client does not read `.env` files by default, so you must run the following command to install `dotenv-cli` locally:
+### Steps
+1. Install `dotenv-cli`.
+  ```terminal
+  npm install dotenv-cli
+  ```
+2. Add the Data Proxy connection string to the `.env` file. Also, comment out the direct connection string.
 
-```
-npm install dotenv-cli
-```
+  ```bash file=.env
+  # DATABASE_URL="postgres://..."
+  DATABASE_URL="prisma://..."
+  ```
+
+### Result
+
+The configuration of your local environment is now ready to send Prisma queries to the database through the Data Proxy.
+
+### What's next
 
 You are now ready to test your script locally.
 
-## 8. Test your script locally
+## 7. Test your application locally
 
-Execute your script locally:
+You can now start your application locally and test the creation of log entries.
 
-```terminal
-npx dotenv -- deno run -A ./index.ts
-```
+### Steps
 
-In a web browser, open [http://localhost:8000/](http://localhost:8000/). This page writes your request to the database.
+1. Run your script locally. 
+  ```terminal
+  npx dotenv -- deno run -A ./index.ts
+  ```
+2. In a web browser, open [http://localhost:8000/](http://localhost:8000/). This page writes your request to the database.
 
-```
-{
-  "id": 3,
-  "level": "Info",
-  "message": "GET http://localhost:8000/",
-  "meta": {
-    "headers": "{}"
+  ```
+  {
+    "id": 3,
+    "level": "Info",
+    "message": "GET http://localhost:8000/",
+    "meta": {
+      "headers": "{}"
+    }
   }
-}
-```
+  ```
+2. Reload the page a few times.<br /><br />Every time you reload, the script generates a new log entry and the id of the current log entry increments. If you open the [Data Browser](/data-platform/data-browser) for your project in Prisma Data Platform, you can verify that the application writes `Info` level logs to your database.
+
+  ![Prisma Data Browser](../../../doc-images/data-platform/data-browser/data-browser-simple-app-data.png)
 
-Refresh the page a couple times to verify that it's working. Now if you click on the **Data Browser** tab on your Prisma Cloud project, you should see `Info` logs written to your database.
+### Result
 
-![Prisma Data Browser](./450-06-data-browser.png)
+This confirms that your application works when you run it from your local environment.
 
-It's working locally!
+## 8. Create a repository and push to GitHub
+
+Create a private repository for your application on GitHub. You need the GitHub repository to add your project to Deno Deploy and enable automated deployments on push.
+
+<Admonition>
+
+**Note**<br /><br />
+
+Optionally, you can also [link your GitHub repository to your project](/data-platform/environments/create#projects-with-an-introspected-schema-or-with-no-schema) in the Prisma Data Platform so that you can keep it up-to-date with the latest changes you make in the `schema.prisma` file.
+
+</Admonition>
```

Comment:
Maybe move this admonition _after_ these instructions? Chronologically this can only happen after it is on GitHub.

Reply:
Done.